wordpress-mobile/WordPress-Android
# Fix reader launch on attachment view

Fixes #13244 

This PR adds support for viewing post types of Attachments via the browser, so that the user is not presented with a reader error view.


Merge instructions
- This PR includes a commit hash for FluxC [PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1737). Once this PR is tested and approved, the FluxC PR can be merged. Then the build.gradle file in this PR has to be updated with the FluxC tag (iirc) and pushed before the branch can be merged. It might be me who does all this after the approve, but I'm not 100% sure, so please let me know. Thanks. 

**To test:**
- Create a new post
- Upload an image to the post, make sure the link is set to “Attachment Page”
- Publish the post and visit it logged out so stats are bumped
- Click on the image so it takes you to the attachment page
- Check stats for the day. 
- You should see the attachment page (a.k.a. image name) in the top posts/pages - tap on this row to open the details as seen in 
- Tap on the header row (Showing stats for - should be the name of your image)
- The image should open in the browser and not in the reader

PR submission checklist:
